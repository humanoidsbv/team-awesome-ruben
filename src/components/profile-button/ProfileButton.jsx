import React from 'react';

import IconArrowDown from '../../assets/icons/icon-arrow-down.svg';
import LogoHumanoids from '../../assets/icons/logo-humanoids.svg';
import UserProfileImage from '../../../static/images/picture-tnt.jpg';
import styles from './ProfileButton.module.css';

const ProfileButton = () => (
  <button type="button" className={styles.user}>
    <LogoHumanoids className={styles.logo} />
    <img
      alt="Profile avatar"
      className={styles.profileImage}
      src={UserProfileImage}
    />
    <IconArrowDown className={styles.arrow} />
  </button>
);

export default ProfileButton;
