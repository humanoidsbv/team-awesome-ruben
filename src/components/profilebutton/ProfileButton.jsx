import React from 'react';
import styles from './ProfileButton.module.css';

import IconArrowDown from '../../assets/icons/icon-arrow-down.svg';
import LogoHumanoids from '../../assets/icons/logo-humanoids.svg';
import UserProfileImage from '../../../static/images/picture-tnt.jpg';

const ProfileButton = () => (
  <button type="button" className={styles.user}>
    <LogoHumanoids className={styles.logo} />
    <img
      alt="ProfileImage"
      className={styles.profileImage}
      src={UserProfileImage}
    />
    <IconArrowDown className={styles.arrow} />
  </button>
);

export default ProfileButton;
