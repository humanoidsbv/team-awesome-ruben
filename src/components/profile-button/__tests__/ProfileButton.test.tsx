import * as React from 'react';
import { shallow } from 'enzyme';

import ProfileButton from '../ProfileButton';

it('matches component output with expected output', () => {
  const profileButton = shallow(<ProfileButton />);
  expect(profileButton).toMatchSnapshot();
});
