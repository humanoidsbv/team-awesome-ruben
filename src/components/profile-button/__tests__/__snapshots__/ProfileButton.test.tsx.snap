// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches component output with expected output 1`] = `
<button
  className="user"
  type="button"
>
  <test-file-stub
    className="logo"
  />
  <img
    alt="Profile avatar"
    className="profileImage"
    src="test-file-stub"
  />
  <test-file-stub
    className="arrow"
  />
</button>
`;
